<launch>

    <include file="$(find mrover)/launch/esw_generic.launch" />
    <node name="cache_bridge" pkg="mrover" type="cache_bridge" />
    <node name="sa_bridge" pkg="mrover" type="sa_bridge" />
    <node name="science_bridge" pkg="mrover" type="science_bridge" />
    <!-- TODO - FIGURE OUT HOW TO ADD THE DIRT SENSOR NODE -->

</launch>